examples/openvino/question-answering/run_qa.py

```diff
@@ -45,7 +45,9 @@
 from transformers.utils.versions import require_version
 
 import evaluate
-from optimum.intel.openvino import OVConfig
+import jstyleson as json
```

Comment:
Is this extra requirement necessary? It seems it's only to load the non-standard json, and shouldn't nncf's safe_open do that already?

Reply:
@helena-intel `safe_open` is only responsible for opening the file. The reason of using non-standard json is that, as a user myself, I find that I occasionally put some comments in the json, it is a flexibility during experiments. It was inspired by nncf out-of-the-box vision examples which use the lib `jstyleson`.